hyperbook/book/spiele/penguins/xml.md

```diff
@@ -131,8 +131,18 @@ Der Inhalt der `field` Objekte steht für die Anzahl Fische auf dem Feld, oder f
 
 `y`: `<list>` index in `<board>`
 
+##### Koordinatensysteme
+
 Beachtet, dass der XML-Status [odd-r Koordinaten](https://www.redblobgames.com/grids/hexagons/#neighbors-offset) nutzt, und dass die Koordinaten in Zügen als [doubled Koordinaten](https://www.redblobgames.com/grids/hexagons/#neighbors-doubled) interpretiert werden.
 
+Die Formeln für die Transformation von dem einen System in das andere sind folgendermaßen:
+
+odd-r zu doubled: ``(x, y) => (x * 2 + y % 2, y)``
+
+doubled zu odd-r: ``(x, y) => (x / 2 - y % 2, y)``
```

Comment:
It should perhaps be noted here that the result must be rounded up?

Reply:
up? Not down?
In the example here it looks like the first row should be rounded to X=0
![image](https://user-images.githubusercontent.com/19777592/187049235-bc67dbe1-3658-4f82-a0e1-c7b0179bca34.png)
Anyway most programming languages automatically round down for integers. Nonetheless adding a comment could be useful.

Reply:
Btw ist hier umgesetzt 
https://github.com/software-challenge/docs/pull/66/commits/0a71d4edae2bbb332fc6d8d02eb6284d3911b198

Reply:
Well the problem that occurs with that is, whe you have for instance the coordinate `(3, 3)`.
The calculation would be: `(3, 3)->(3 / 2 - 3 % 2, 3)->(0.5, 3)` and that would be `(0, 3)` when cast into integer, what is not correct.

Reply:
Ahh right, I didnt think about the -1
I guess Ill have to fix some code later then